### PR TITLE
[button] fix: remove onBlur that changed target checked value

### PR DESCRIPTION
**Pull request by maimonav**
`onBlur` event changed target checked value, fix it by removing it.

### CODE REVIEW

File: packages/plugin-button/web/src/components/settings-component.jsx [hunk 1]
```diff
@@ -41,10 +41,6 @@ class SettingsComponent extends PureComponent {
     this.setState({ rel: event.target.checked });
   };
 
-  onBlur = event => {
```
Comment (shaulgo) [inline packages/plugin-button/web/src/components/settings-component.jsx:44]: Very strange. Any ideas why we had this?

**Review by shaulgo — APPROVED** [review]
Make sure this gives the result we want. This is very strange.

**Comment by shaulgo** [comment]
test is failing for link button

File: packages/plugin-button/web/src/components/settings-component.jsx (hunk 1 above)
Comment (maimonav) [reply packages/plugin-button/web/src/components/settings-component.jsx:44]: Indeed strange, I have no idea why it was implemented like that in the first place.
I don't see any reason why we need to set the checked value of the radio button to false onBlur.

**Comment by maimonav** [comment]
I'm updating the test snapshot now, the change is in the contentState, target changed from `false` to `true`.

**Pull request merged**